pkgs/development/interpreters/emilua/default.nix

```diff
@@ -89,8 +89,7 @@ stdenv.mkDerivation (self: {
   dontUseCmakeConfigure = true;
 
   mesonFlags = [
-    (lib.mesonBool "enable_file_io" true)
-    (lib.mesonBool "enable_io_uring" true)
+    (lib.mesonBool "enable_io_uring" false)
```

Comment:
Why though?

Reply:
file_io breaks build and uring breaks tests

uring, at least in my machine, doesn't work in the nix sandbox

Reply:
BTW I was in a back and forth with upstream to try to fix it. Previously it failed some of the tests, now all of them fail because it checks for uring in initialization time.

Reply:
Should we add a comment for that?

Reply:
I made it opt-in.

Had to disable by default because uring is not allowed in the Nix sandbox, at least in my machine, and it breaks tests, so if uring is enabled the tests are disabled.